tests/plugins/pastefromword/generated/_helpers/pfwTools.js

```diff
@@ -90,14 +90,15 @@
 			} ),
 
 			// Firefox adds quotes around all fonts in font-family.
+			// Safari replaces `&guot;` with `'` in quoted font names.
```

Comment:
I not sure if it is a bug, the specs allows usage of double `"` or single quotes `'` for font names. I can report it if we would like to keep a reference.

You may see [this codepen](https://codepen.io/f1ames/pen/NjxrMN?editors=1010) in both Chrome and Safari - the latter one replaces `"` with `'` in font-family property.